actors/miner/tests/util.rs

```diff
@@ -2490,13 +2491,13 @@ where
 
 // Returns a fake hashing function that always arranges the first 8 bytes of the digest to be the binary
 // encoding of a target uint64.
-fn fixed_hasher(offset: ChainEpoch) -> Box<dyn Fn(&[u8]) -> [u8; 32]> {
-    let hash = move |_: &[u8]| -> [u8; 32] {
-        let mut result = [0u8; 32];
+fn fixed_hasher(offset: ChainEpoch) -> Box<dyn Fn(SupportedHashes, &[u8]) -> ([u8; 64], usize)> {
```

Comment:
Hm. What's the max digest length in the EVM?

Reply:
32 bytes, but our hash function can do up to 64 bytes